modules/nf-core/gatk4/cnnscorevariants/main.nf

```diff
@@ -3,7 +3,7 @@ process GATK4_CNNSCOREVARIANTS {
     label 'process_low'
 
     //Conda is not supported at the moment: https://github.com/broadinstitute/gatk/issues/7811
```

Comment:
Is conda fixed or still not then?

Reply:
haven't tested 🤷🏻  want to focus on getting the latest version available first

Reply:
Just tested it, no dice, still issues with the python libs not being found